File: WebProdavnica/src/model/DodatnaUsluga.java
package model;

import java.io.Serializable;

public class DodatnaUsluga implements Serializable {

	/**
	 * 
	 */
	private static final long serialVersionUID = -2819206760986121403L;
	protected String naziv;
	protected String opis;
	protected double cena;
	
	public DodatnaUsluga(){}

	
	public DodatnaUsluga(String naziv, String opis, double cena) {
		super();
		this.naziv = naziv;
		this.opis = opis;
		this.cena = cena;
	}
	


	public String getNaziv() {
		return naziv;
	}


	public void setNaziv(String naziv) {
		this.naziv = naziv;
	}


	public String getOpis() {
		return opis;
	}


	public void setOpis(String opis) {
		this.opis = opis;
	}


	public double getCena() {
		return cena;
	}


	public void setCena(double cena) {
		this.cena = cena;
	}


	@Override
	public String toString() {
		return "DodatnaUsluga [naziv=" + naziv + ", opis=" + opis + ", cena="
				+ cena + "]";
	}
	@Override
	public boolean equals(Object obj) {
		if(obj==null)
			return false;
		DodatnaUsluga dodatnaUsluga = (DodatnaUsluga)obj;
		if(getNaziv().trim().equalsIgnoreCase(dodatnaUsluga.getNaziv().trim()))
			return true;
		else return false;
	}
}

File: WebProdavnica/WebContent/js/dodatnaUsluga.js
/**
 * 
 */
function DodatnaUsluga(){
	this.naziv;
	this.opis;
	this.cena;

	DodatnaUsluga.prototype.setNaziv = function(naziv) {
		this.naziv = naziv;
	}	
	DodatnaUsluga.prototype.getNaziv = function() {
		return this.naziv;
	}
	DodatnaUsluga.prototype.setOpis = function(opis) {
		this.opis = opis;
	}
	DodatnaUsluga.prototype.getOpis = function() {
		return this.opis;
	}
	DodatnaUsluga.prototype.setCena = function(cena) {
		this.cena = cena;
	}
	DodatnaUsluga.prototype.getCena = function() {
		return this.cena;
	}
	DodatnaUsluga.prototype.fillData = function(naziv,opis,cena) {
		this.naziv =$(naziv).val();
		this.opis =$(opis).val();
		this.cena =$(cena).val();
		console.log(this.naziv + ' - ' + this.cena);
		$.ajax({
			dataType: 'json',
			contentType: 'application/json',
			mimeType: 'application/json',
			url: 'AddDodatnaUsluga',
			type: 'POST',
			data: JSON.stringify(dodatnaUsluga),
			success: function(data) {
				console.log(data);
				window.location = data.url;
			},
			error: function(data,status,er) {
	            alert("error: " + data + " status: " + status + " er:" + er);
	        }
		});
	}
	DodatnaUsluga.prototype.fillDataEdit = function(naziv,opis,cena) {
		this.naziv =$(naziv).val();
		this.opis =$(opis).val();
		this.cena =$(cena).val();
		console.log(this.naziv + ' - ' + this.cena);
		$.ajax({
			dataType: 'json',
			contentType: 'application/json',
			mimeType: 'application/json',
			url: 'EditDodatnaUsluga',
			type: 'POST',
			data: JSON.stringify(dodatnaUsluga),
			success: function(data) {
				console.log(data);
				window.location = data.url;
			},
			error: function(data,status,er) {
	            alert("error: " + data + " status: " + status + " er:" + er);
	        }
		});
	}
	DodatnaUsluga.prototype.searchDU = function(polje,vrednost) {
		this.naziv =$(naziv).val();
		this.opis =$(opis).val();
		this.cena =$(cena).val();
		console.log(this.naziv + ' - ' + this.cena);
		$.ajax({
			dataType: 'json',
			contentType: 'application/json',
			mimeType: 'application/json',
			url: 'EditDodatnaUsluga',
			type: 'POST',
			data: JSON.stringify(dodatnaUsluga),
			success: function(data) {
				console.log(data);
				window.location = data.url;
			},
			error: function(data,status,er) {
	            alert("error: " + data + " status: " + status + " er:" + er);
	        }
		});
	}
}

var dodatnaUsluga = new DodatnaUsluga();